sdk/modelsrepository/azure-iot-modelsrepository/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### New features
+
+- N/A
```

Comment:
We have nothing to say here?

Reply:
We do, I will update this as we get closer to release.